server/server_test.go

```diff
@@ -240,15 +230,15 @@ func TestPublish(t *testing.T) {
 				status, ok := status.FromError(err)
 				require.True(t, ok, "expected gRPC error")
 				require.Equal(t, codes.InvalidArgument, status.Code())
-				require.Equal(t, tc.expectedMsg, status.Message())
+				require.Contains(t, status.Message(), tc.expectedMsg)
```

Comment:
This is to fixing https://github.com/elastic/elastic-agent-shipper/pull/90#discussion_r944688097